pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -74,6 +74,10 @@ const (
 	// ServiceAnnotationAllowedServiceTag is the annotation used on the service
 	// to specify a list of allowed service tags separated by comma
 	ServiceAnnotationAllowedServiceTag = "service.beta.kubernetes.io/azure-allowed-service-tags"
+
+	// ServiceAnnotationLoadBalancerIdleTimeout is the annotation used on the service
+	// to specify the idle timeout for connections on the load balancer.
```

Comment:
The timeout is in seconds?

nit: s/the idle timeout/the idle timeout in seconds/